src/SingleBookingPage/bookingItem/Accordion.js

```diff
@@ -61,43 +63,51 @@ const Accordion = ({ trip }: Props) => {
   const arrivalCode = idx(trip, _ => _.arrival.airport.locationId);
 
   return (
-    <div className="AccordionWrapper">
-      <Box padding="12px 16px 16px 12px">
-        <div className="Accordion">
-          <div className="header">
-            <div className="logo">
-              <CarrierLogoWrapper legs={trip.legs} />
-            </div>
-            <div className="info">
-              <Typography type="secondary" size="small">
-                {departureDate && formatDate(departureDate)}
-              </Typography>
-              <div className="itinerary">
-                <span className="location">
-                  <Typography variant="bold" type="primary">
-                    {departureCity}&nbsp;({departureCode})
-                  </Typography>
-                </span>
-                <span className="arrow">
-                  <span className="inline-icon">
-                    <AirplaneRight />
-                  </span>
-                </span>
-                <span className="location">
-                  <Typography variant="bold" type="primary">
-                    {arrivalCity}&nbsp;({arrivalCode})
-                  </Typography>
-                </span>
+    <LanguageContext.Consumer>
```

Comment:
Maybe making the whole `Accordion` a consumer of Language is overkill. Also other components will most certainly need to format dates.

We could change the approach here and make our utils function, ie. `formatDate` the `Consumer` of our context, since this is where it's needed after all. What do you think? :slightly_smiling_face:

Reply:
Yes, I have thought about it, but I wasn't sure about the better approach if coupling the helper with `LanguageContext` or import this in all components when we need. But if we have others components, I agree with you. I'm going to change it. 👨‍💻